src/components/BulkOperations/index.jsx

```diff
@@ -0,0 +1,194 @@
+import  { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import Papa from 'papaparse';
+
```

Comment:
Separate package imports and project internal imports with a blank line to keep the code clean